docs/topics/templates/document-attributes.adoc

```diff
@@ -173,3 +173,6 @@
 
 // WildFly Swarm repository (mostly for the Contribution Guide)
 :link-repo-wildfly-swarm: https://github.com/wildfly-swarm/wildfly-swarm/
+
+// Link S2I process (primarily for guides mission interaction sections)
+:link-s2i-process: https://docs.openshift.com/container-platform/3.4/architecture/core_concepts/builds_and_image_streams.html#source-build
```

Comment:
Can you update all the links to OpenShift docs to the latest version? These links go to 2.4 which is very out-of-date. You can use "latest".
https://docs.openshift.com/container-platform/latest/architecture/core_concepts/builds_and_image_streams.html#source-build

Reply:
@astrong1 yes I'll update. Thanks

Reply:
@astrong1 @rhoads-zach  Did you see the Updates done per your suggestions.